doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown

```diff
@@ -0,0 +1,36 @@
+YOLO DNNs  {#tutorial_dnn_yolo}
+===========================
+
+Introduction
+------------
+
+In this text you will learn how to use opencv_dnn module using yolo_object_detection (Sample of using OpenCV dnn module in real time with device capture, video and image).
+
+We will demonstrate results of this example on the following picture.
+![Picture example](images/yolo.jpg)
+
+Examples
+-----------
+
+For compile and use:
+
+VIDEO DEMO:
+@youtube{NHtRlndE2cg}
+
+How to compile in comand line:
+
+```$ g++ `pkg-config --cflags opencv` `pkg-config --libs opencv` yolo_object_detection.cpp -o yolo_object_detection```
```

Comment:
Try
```
    @code{.bash}
    g++ `pkg-config --cflags opencv` `pkg-config --libs opencv` yolo_object_detection.cpp -o yolo_object_detection
    @endcode
```

More examples of code blocks:
https://raw.githubusercontent.com/opencv/opencv/master/doc/tutorials/introduction/linux_install/linux_install.markdown